README.md

```diff
@@ -1,6 +1,18 @@
 # HACKTOBERFEST 2020
 
 ## How to Contribute?
+
+### Short Method
```

Comment:
This method has been written here so that students can learn about how to work with Git and GitHub